quickstarts/aws/aws-apprunner/config.yml

```
@@ -0,0 +1,25 @@
+slug: aws-apprunner
+description: |
+  ## Monitor your Apps running in App Runner coutainers on New Rerilc Instant Observability Platform!
+  With New Relic's AWS App Runner integration, you now can see all your App Runner data on New Rerilc Instant Observability Platform.
+  ## Get started!
+  Check out the [documentation](https://docs.newrelic.com/docs/integrations/amazon-integrations/aws-integrations-list/aws-apprunner) to learn more about New 
+Relic monitoring and integration with AWS App Runner.
+summary: |
+  Use New Relic - AWS App Runner Integration to Monitor Your Conteinerized Apps and Services with New Relic
+icon: logo.svg
+level: New Relic
+authors:
+  - New Relic
+title: Amazon App Runner with New Relic
+documentation:
+  - name: Amazon App Runner integration docs
+    description: |
+      Faster and comprehensive telemetry from Amazon App Runner Service for new and existing New Relic Customers
+    url: >-
+      https://docs.newrelic.com/docs/integrations/amazon-integrations/aws-integrations-list/aws-apprunner
+keywords:
+  - aws
+  - App Runner
```

```suggestion
  - app runner
  - logs
```

the search is case sensitive so I usually leave these lower case, also adding logs will help place this into a category.